config/README.md

```diff
@@ -40,10 +40,22 @@ items_filepath: (optional) absolute filepath to the 07_Items.txt plaintext docum
 
 basic_rules_filepath: (optional) absolute filepath to the Basic Rules.txt plaintext document from the compound_x repo
 
+combat_rules_filepath: (optional) absolute filepath to the 01_02_Combat_Rules.txt plaintext document in teh compound_x repo
+
+conditions_filepath: (optional) absolute filepath to the conditions 01_05_Conditions.txt plaintext file
+
+glossary_filepath: (optional) absolute filepath to the 03_Glossary_of_Terms.txt file
+
+cloaking_filepath: (option) absolute filepath to the cloaking rules file
```

Comment:
Nonblocker: This one says "option" instead of "optional" like all the others. Problem?